src/Compilers/CSharp/Test/Emit/Attributes/EmitTestStrongNameProvider.cs

```diff
@@ -1,11 +1,16 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
are u changing the BOM on these files? If so we should revert

Reply:
VS autoformat was doing this.  Not sure why.